pkg/envoy/boilerplate_test.go

```diff
@@ -0,0 +1,21 @@
+package envoy
+
+import (
+	"testing"
+
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/route"
+	hcm "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/http_connection_manager/v2"
+)
+
+func TestTracingEnabledOnFilter(t *testing.T) {
+	var virtualHosts []route.VirtualHost
+	httpConnectionManager := makeConnectionManager(virtualHosts)
+
+	if httpConnectionManager.Tracing == nil {
+		t.Error("expected tracing configuration to exist, was nil")
+	}
+
+	if httpConnectionManager.Tracing.OperationName != hcm.EGRESS {
+		t.Errorf("expected tracing operation name to be EGRESS, was %v", httpConnectionManager.Tracing.OperationName)
+	}
+}
```

Comment:
this test seems a bit superfluous, you're just testing that something you've hardcoded is there. if we were dynamically turning tracing on and off on a per listener basis it would be more valuable.